routes/api/category-routes.js

```diff
@@ -1,28 +1,93 @@
-const router = require('express').Router();
-const { Category, Product } = require('../../models');
+const router = require("express").Router();
+const { Category, Product } = require("../../models");
 
 // The `/api/categories` endpoint
 
-router.get('/', (req, res) => {
-  // find all categories
-  // be sure to include its associated Products
+router.get("/", (req, res) => {
+  Category.findAll({
+    include: {
+      model: Product,
```

Comment:
I really like that you have chosen to return all of the products belonging to a category - this is a very pleasant addition for the user